### DISCUSSION TITLE
How to get Galaxy Brain Badge?

### DISCUSSION THREAD

**Discussion by xRodzMC** [community/community#27808] [Other Feature Feedback, Questions, & Ideas] [answered]
![](https://github.githubassets.com/images/modules/profile/achievements/galaxy-brain-default.png)
Can someone help me to get this badge?

**Comment by jjaaster** [comment]
@xqwtxon to earn this achievement Galaxy brain you have to mark someone's answer correct on a discussion in order to earn it

**Reply by rofid0ank** [reply]
Thanks, for information ^_^

**Reply by AmniX** [reply]
Nice Info.

**Reply by haritsrm** [reply]
Cool

**Reply by WuglyakBolgoink** [reply]
nice )

**Reply by EliasAfara** [reply]
Hmm interesting, but I don't think it works that way.

**Reply by HarikrishnanBalagopal** [reply]
Hmm, weird

**Reply by xRodzMC** [reply]
Thank you <3

**Reply by CostaMateus** [reply]
thx

**Reply by baraaj** [reply]
Good

**Reply by dtherhtun** [reply]
Cool

**Comment by radar-blip** [comment]
You have to have 2 accepted answers to a discussion to unlock the Galaxy Brain achievement badge.
Here is all the info you need: https://github.com/Schweinepriester/github-profile-achievements#achievements

**Comment by EliasAfara** [comment]
> Can someone help me to get this badge?

Hi, to get the Galaxy Brain achievement badge, you need to fulfill the following criteria:

Discussions should be in a public repository
Get marked as answers to 2 or more questions
The questions cannot be asked by you. In short, you cannot answer your own questions, those do not count.
There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.

Reference: https://github.com/orgs/community/discussions/20950#discussioncomment-3190178
Hope these steps help you in achieving your badge! Good luck :)

More details here:
https://github.com/Schweinepriester/github-profile-achievements

**Comment by jfullstackdev** [comment]
other than marking answers by others, you need to refrain from marking your own answers or you will not get that badge

**Reply by Lszinndev** [reply]
.

**Comment by iamfoysal** [comment]
@jjaster10, to achieve the 'Galaxy Brain' achievement, take note of the following:

Answer at least 2 discussions.
Take note, you CAN NOT mark your own answers to your own questions. They have to be marked by another user.
Discussions should be in a public repository.
There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.
Hope this answer has helped you, and all the best in getting the achievement! 

Reference: https://github.com/orgs/community/discussions/18467

**Comment by Qeyf** [comment]
You have to have 2 accepted answers to a discussion to unlock the Galaxy Brain achievement badge.
Here is all the info you need: https://github.com/Schweinepriester/github-profile-achievements#achievements

**Reply by baraaj** [reply]
How does it work please

**Comment by baraaj** [comment]
you just need to mark someone's answer correct on a discussion

**Reply by radar-blip** [reply]
Its not about marking someones answer. Its about your answer being accepted.

**Comment by Veercodeprog** [comment]
to achieve the 'Galaxy Brain' achievement, take note of the following:

Answer at least 2 discussions.
Take note, you CANNOT mark your own answers to your own questions. They have to be marked by another user.
Discussions should be in a public repository.
There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.

**Comment by github-staff** [comment, marked as answer]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.

**Comment by Hazrat-Ali9** [comment]
How do I get a Galaxy Brain badge
Response
